frontend/src/layouts/AddressSettings.vue

```diff
@@ -8,22 +8,30 @@
         <span v-if="userDisplayName">
           {{ userDisplayName }}
         </span>
-        <Jazzicon v-if="userAddress" :address="userAddress" class="q-ml-sm" />
+        <span v-if="userAddress">
+          <Avatar v-if="avatar" :avatar="avatar" class="q-ml-sm" />
+          <Jazzicon v-else :address="userAddress" class="q-ml-sm" />
```

Comment:
What do you think about deleting the `Jazzicon.vue` component and having a single `Avatar.vue` component, to reduce the number of files? Arguably overkill right now, but anywhere we want to show the user's image we'd now need this logic, so we can simplify it with a single component that takes address or avatar and shows the right image accordingly

Reply:
I like that approach, good call 👍